modules/acquia_cms_site_studio/acquia_cms_site_studio.module

```diff
@@ -213,3 +214,20 @@ function acquia_cms_site_studio_rebuild_styles() {
   // rebuild.
   batch_set(install_acms_site_studio_rebuild());
 }
+
+/**
+ * Implements hook_modules_installed().
+ */
+function acquia_cms_site_studio_modules_installed($modules) {
+  // Set default node revision delete configuration for Place content type.
```

Comment:
@rajeshreeputra - Update comment here, it should be page not place

Reply:
updated